tests/unit/models/test_ledger_object.py

```diff
@@ -0,0 +1,976 @@
+from unittest import TestCase
+
+from xrpl.models import IssuedCurrencyAmount
+from xrpl.models.auth_account import AuthAccount
+from xrpl.models.currencies.issued_currency import IssuedCurrency
+from xrpl.models.currencies.xrp import XRP
+from xrpl.models.ledger_objects import (
+    AMM,
+    DID,
+    AccountRoot,
+    Amendments,
+    Check,
+    DepositPreauth,
+    DirectoryNode,
+    DisabledValidator,
+    Escrow,
+    FeeSettings,
+    LedgerHashes,
+    Majority,
+    NegativeUNL,
+    NFToken,
+    NFTokenOffer,
+    NFTokenPage,
+    Offer,
+    PayChannel,
+    RippleState,
+    SignerEntry,
+    SignerList,
+    Ticket,
+)
+from xrpl.models.ledger_objects.amm import AuctionSlot, VoteEntry
+from xrpl.models.ledger_objects.bridge import Bridge
+from xrpl.models.ledger_objects.xchain_owned_claim_id import (
+    XChainClaimProofSig,
+    XChainOwnedClaimID,
+)
+from xrpl.models.ledger_objects.xchain_owned_create_account_claim_id import (
+    XChainCreateAccountProofSig,
+    XChainOwnedCreateAccountClaimID,
+)
+from xrpl.models.xchain_bridge import XChainBridge
+
+account_root_json = {
+    "account": "rf1BiGeXwwQoi8Z2ueFYTEXSwuJYfV2Jpn",
```

Comment:
These objects are rarely going to be constructed themselves, they're usually going to be taken from RPCs. Could we instead test the normal XRPL formatting with `from_xrpl`, since that's how it's most commonly going to be used?

Reply:
Bump

Reply:
I already replaced the tests to use `from_xrpl` and they pass.

Reply:
That's strange, they shouldn't be passing... You should need CamelCase keys for `from_xrpl` to pass.

Reply:
I just switched the JSONs to CamelCase but it looks like they pass for both CamelCase and snake_case 😅

Reply:
That should really be fixed.... But that can be a separate PR.

Reply:
Created an issue for it - https://github.com/XRPLF/xrpl-py/issues/689